From d3a3b279455a4928a6279e82e3ca85abb0e8fa5c Mon Sep 17 00:00:00 2001
From: Francesco Giancane <francesco.giancane@polito.it>
Date: Mon, 18 Sep 2017 15:40:05 +0200
Subject: [PATCH] Fixed import name.

---
 alexapi/AlexaPi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alexapi/AlexaPi b/alexapi/AlexaPi
index a8c1cd8..7462242 100755
--- a/alexapi/AlexaPi
+++ b/alexapi/AlexaPi
@@ -13,7 +13,7 @@ import json
 import optparse
 import email
 import subprocess
-from future.builtins import bytes
+from __builtin__ import bytes
 
 import yaml
 import requests
-- 
2.13.5

